app/views/users/registrations/_mentee_form.html.erb

```diff
@@ -0,0 +1,47 @@
+<div>
+  <div>
+    <h3>Mentee</h3>
+    <%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name), defaults: {input_html: {class: "subForm"}}) do |f| %>
+      <%= f.error_notification %>
+      <div class="form-inputs">
+        <%= f.input :email,
+                required: true,
+                autofocus: true,
+                input_html: { autocomplete: "email"}%>
+        <%= f.input :password,
+                required: true,
+                hint: ("#{@minimum_password_length} characters minimum" if @minimum_password_length),
+                input_html: { autocomplete: "new-password"} %>
+        <%= f.input :password_confirmation,
+                required: true,
+                input_html: { autocomplete: "new-password"} %>
+        <%= f.input :first_name,
+                required: true %>
+        <%= f.input :last_name,
+                required: true %>
+        <%= f.input :phone_number,
+                required: true %>
+        <%= f.input :bio,
+                required: true %>
+        <%= f.input :timezone, collection: ActiveSupport::TimeZone.all.map { |tz| [tz.name, tz.name] },
+             required: true, label: 'Select Timezone' %>
+        <%= f.input :title, label: 'Job Title',
+                required: true %>
+        <%= f.input :linkedin_link,
+                required: true %>
+        <%= f.input :profile_picture,
+                required: true %>
+        <%= f.input :skills_array,
+                required: true, multiple: true %>
+      </div>
+      <%= f.simple_fields_for :cohorts, CohortMember.new do |cm| %>
+        <%= cm.hidden_field :capacity, value: nil %>
+        <%= cm.hidden_field :role, value: 'Mentee' %>
+        <%= cm.hidden_field :cohort_id, value: 1 %>
```

Comment:
Just wondering - will this continuously put every new Mentee into Cohort 1 no matter what?